src/source-control/GitSourceControl.ts

```diff
@@ -198,6 +198,33 @@ export class GitSourceControl implements SourceControl {
     this.repo = ourRepository;
   }
 
+  private getLLBFromConfig(): BranchName[] {
```

Comment:
Nit: this acronym is a little confusing hahah

```suggestion
  private getLongLivedBranchesFromConfig(): BranchName[] {
```